crates/matrix-sdk-ui/src/notification_client.rs

```diff
@@ -573,7 +573,8 @@ impl NotificationClientBuilder {
     }
 }
 
-enum RawNotificationEvent {
+#[derive(Debug)]
+pub enum RawNotificationEvent {
```

Comment:
Since this is now public, can you add a doc comment explaining what it is, and comment each variant too, please?

Reply:
@bnjbvr the other (public) `NotificationEvent` didn't have docs either, so I added them for both ...